structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,246 @@
+
+It also exists to serve as a coordination point for events and other cross-community projects within the Minecraft sphere.
+
+## What the Program is not
+
+* An official, direct affiliation with the Quilt project, or any other community that participates in the Program
+* A space for building or coordinating the character assassinations of community members or prominent personalities
+* A space meant for blindly synchronizing punishments between communities
+  * Instead, each community must decide on appropriate moderation actions for their own spaces, which may be informed by the data they're given access to, but not blindly copied between communities --- Breaking this rule is likely to result in a community being removed from the Program
+
+## Program Roles
+
+As the Program exists largely on Discord, the roles of its members are directly represented by Discord server roles.
+
+### Keyholder
+
+The Keyholder for this space is the same as for the other Quilt spaces. This role is described in the Keyholders section of [RFC 7: Community Team](https://github.com/QuiltMC/rfcs/blob/master/structure/0007-community-team.md#keyholders), and works in the same way.
+
+### Managers
+
+Managers are responsible for keeping the Program up-and-running. They take care of the following tasks, among other things:
+
+* Maintaining, organizing, and actioning changes to the Program's Discord server, and any other Program-related spaces
+* Organizing polls and votes, as required by formal procedures
+* Aiding in conflict resolution between communities and Program members, and acting as moderators of Program spaces when necessary
+* Staying in touch (and maintaining relations) with the communities that are part of the Program
+* Procuring and maintaining any services that the Program needs to function properly
+
+This role is very similar to the Community Manager role within Quilt's own Community Team.
+
+### Collaborators
+
+The Collaborator role is given to moderators from the communities that are part of the Program, and who have joined a Program-related space in order to represent their communities. Administrators from these communities are welcome to join as Collaborators as well, if they wish.
+
+Collaborators are given access to all moderation-related areas within Program spaces, such as general moderation chat channels and case-tracking threads. Additionally, all Collaborators are granted access to spaces dedicated to organizing events between participating communities.
+
+### Individual Collaborators
+
+The Individual Collaborator role is given to moderators that have joined the Program, but do not represent any specific community. Individual Collaborators can make use of the Program and contribute to it just as regular Collaborators can.
+
+Individuals are given the same level of access that regular Collaborators are given. However, Individual Collaborators are not given veto rights but not voting rights.
```

Comment:
```suggestion
Individuals are given the same level of access that regular Collaborators are given, including veto rights but not voting rights.
```

Reply:
voting rights are not "levels of access" or even "access" at all